src/views/Layout.jsx

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
We can also remove this lint rule override since we've implemented the `NavLink` component! To fully address the lint rule, we should remove the `<a>` tags on line 25.